Wiki.py

```diff
@@ -0,0 +1,44 @@
+import wikipedia
+from vaderSentiment.vaderSentiment import SentimentIntensityAnalyzer
+
+def wikiInfo():
+    cat = wikipedia.page("Cat")
+    dog = wikipedia.page("Dog")
+
+    catInfo = cat.content
+    dogInfo = dog.content
+
+    return catInfo, dogInfo
+
+def sent(info):
+    analyzer = SentimentIntensityAnalyzer()
+    return analyzer.polarity_scores(info)
+
+def topTen(info):
+    infoList = info.split()
+
+    stripped = []
+    for word in infoList:
+        stripped.append(word.strip("/.,?'':;[]"")(%"))
+
+    hist = dict()
+    for c in stripped:
+        hist[c] = hist.get(c, 0) + 1
+
+    listHist = []
+    for key in hist:
+        listHist.append((key, hist[key]))
+
+    return sorted(listHist, key=lambda words: words[1], reverse=True)
+
```

Comment:
You can get a sorted list of dictionary items by its value without converting a dictionary to a list. 

```sorted(dict1, key=dict1.get)```

This code will take a dictionary and return a sorted list of dictionary items by its value.